File: Lab2/app/src/main/java/com/example/firstjavaapp/models/Person.java
package com.example.firstjavaapp.models;

public class Person {
    public int id;
    public String name;
    public String enName;
    public String photo;
    public String enProfession;
    public String description;

    @Override
    public String toString(){
        return this.name + " " + this.enName + " " +this.description;
    }
}

File: Lab_2_NEW/app/src/main/java/com/example/lab_2_new/models/Person.kt
package com.example.lab_2_new.models

class Person {
    var id = 0
    var name: String? = null
    var enName: String? = null
    var photo: String? = null
    var enProfession: String? = null
    var description: String? = null

    override fun toString(): String {
        return name + " " + enName + " " + description
    }
}